lib/config-var.js

```diff
@@ -58,6 +59,9 @@ async function update(pg_pool, app_uuid, app_name, space_name, space_tags, org,
     if(service_data[key]) {
       throw new common.ConflictError(`The environment variable ${key} was set by a service, remove the accompanying service to modify or delete this.`)
     }
+    if(config_vars[key] === null && !config_out[key]) {
+      throw new common.ConflictError(`The environment variable ${key} does not exist and cannot be removed.`)
+    }
   }
```

Comment:
Another when in rome bug I found where you can remove environment variables that never existed in the first place, this now errors if you try and remove an environment variable that never existed.